src/include/distributed/multi_physical_planner.h

```diff
@@ -262,6 +262,8 @@ typedef struct TaskQuery
 	}data;
 }TaskQuery;
 
+typedef struct TupleDestination TupleDestination;
```

Comment:
This PR created 2 warnings in my setup. Both are related to the redefinition of `TupleDestination`

2 of 2:
```
In file included from commands/multi_copy.c:74:
In file included from /Users/hanefi/code/citus/citus/src/include/distributed/local_executor.h:15:
/Users/hanefi/code/citus/citus/src/include/distributed/tuple_destination.h:18:33: warning: redefinition of typedef 'TupleDestination' is a C11 feature [-Wtypedef-redefinition]
typedef struct TupleDestination TupleDestination;
                                ^
/Users/hanefi/code/citus/citus/src/include/distributed/multi_physical_planner.h:265:33: note: previous definition is here
typedef struct TupleDestination TupleDestination;
                                ^
```